pkg/kinds/kinds.go

```diff
@@ -48,10 +55,80 @@ const (
 	ResourceQuota                  Kind = "ResourceQuota"
 	Role                           Kind = "Role"
 	RoleBinding                    Kind = "RoleBinding"
+	RuntimeClass                   Kind = "RuntimeClass"
 	Secret                         Kind = "Secret"
+	SelfSubjectAccessReview        Kind = "SelfSubjectAccessReview"
+	SelfSubjectRulesReview         Kind = "SelfSubjectRulesReview"
 	Service                        Kind = "Service"
 	ServiceAccount                 Kind = "ServiceAccount"
 	StatefulSet                    Kind = "StatefulSet"
+	SubjectAccessReview            Kind = "SubjectAccessReview"
 	StorageClass                   Kind = "StorageClass"
+	TokenReview                    Kind = "TokenReview"
 	ValidatingWebhookConfiguration Kind = "ValidatingWebhookConfiguration"
+	VolumeAttachment               Kind = "VolumeAttachment"
 )
+
+// Namespaced returns whether known resource Kinds are namespaced. If the Kind is unknown (such as CRD Kinds), the
+// known return value will be false, and the namespaced value is unknown. In this case, this information can be
+// queried separately from the k8s API server.
```

Comment:
Can we note any docs links for readers to follow along which are / aren't namespaced? e.g. as reported via `$kubectl api-resources`

Reply:
Done